main/modes/system/intro/introMode.c

```diff
@@ -1441,25 +1453,18 @@ static void playIntro(int64_t elapsedUs)
 static void introSwadgePass(int64_t elapsedUs)
 {
     // draw username
-    typedef struct
-    {
-        int16_t x;
-        int16_t y;
-    } locs_t; // this is stupid
-
-    locs_t locs;
-    locs.x = 25;
-    locs.y = 60;
-
     nameData_t username = *getSystemUsername();
-    char prefix[]       = "Your username is \n";
-    char suffix[]       = "\nIf you don't like it, you can adjust it in the next step.";
-    int namelen         = sizeof(username.nameBuffer);
-    int prelen          = sizeof(prefix);
-    int suflen          = sizeof(suffix);
+    const char prefix[] = "Your username is \n";
+    const char* suffix  = suffixes[getTutorialCompletedSetting()];
+
+    int namelen = sizeof(username.nameBuffer);
+    int prelen  = sizeof(prefix);
+    int suflen  = strlen(suffix);
```

Comment:
I got rid of an intermediate `strcpy()` by using a pointer into `suffixes[]`. That also meant using `strlen()` instead of `sizeof()`.